galaxy.yml

```
@@ -9,7 +9,8 @@ authors:
 
 ### optional but strongly advised
 # short summary of the collection
-
+dependencies:
+  "ansible.netcommon": ">=2.5.0"
```

This has to be 2.5.1 to work
```suggestion
  "ansible.netcommon": ">=2.5.1"
```